packages/remark-stringify/readme.md

```diff
@@ -4,95 +4,263 @@
+**mdast** is the markdown AST that remark uses.
+This is a remark plugin that defines how mdast is turned into markdown.
+
+## When should I use this?
+
+This plugin adds support to unified for serializing markdown.
+You can alternatively use [`remark`][remark-core] instead, which combines
+unified, [`remark-parse`][remark-parse], and this plugin.
+
+You can combine `remark-stringify` with other plugins to add syntax extensions.
+Notable examples that deeply integrate with `remark-stringify` are
+[`remark-gfm`][remark-gfm],
+[`remark-frontmatter`][remark-frontmatter],
+[`remark-math`][remark-math], and
+[`remark-directive`][remark-directive].
+You can also use any other [remark plugin][plugin] before `remark-stringify`.
+
+If you want to handle syntax trees manually, you can use
+[`mdast-util-to-markdown`][mdast-util-to-markdown].
 
 ## Install
 
-This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c):
-Node 12+ is needed to use it and it must be `import`ed instead of `require`d.
-
-[npm][]:
+This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
+In Node.js (version 12.20+, 14.14+, or 16.0+), install with [npm][]:
 
 ```sh
 npm install remark-stringify
 ```
 
+In Deno with [Skypack][]:
+
+```js
+import remarkStringify from 'https://cdn.skypack.dev/remark-stringify@10?dts'
+```
+
+In browsers with [Skypack][]:
+
+```html
+<script type="module">
+  import remarkStringify from 'https://cdn.skypack.dev/remark-stringify@10?min'
+</script>
+```
+
 ## Use
 
+Say we have the following module `example.js`:
+
 ```js
 import {unified} from 'unified'
 import rehypeParse from 'rehype-parse'
 import rehypeRemark from 'rehype-remark'
 import remarkStringify from 'remark-stringify'
 
-unified()
-  .use(rehypeParse)
-  .use(rehypeRemark)
-  .use(remarkStringify, {
-    bullet: '*',
-    fence: '~',
-    fences: true,
-    incrementListMarker: false
-  })
-  .process('<h1>Hello, world!</h1>')
-  .then((file) => {
-    console.log(String(file))
-  })
+main()
+
+async function main() {
+  const file = await unified()
+    .use(rehypeParse)
+    .use(rehypeRemark)
+    .use(remarkStringify, {
+      bullet: '*',
+      fence: '~',
+      fences: true,
+      incrementListMarker: false
+    })
+    .process('<h1>Hello, world!</h1>')
+
+  console.log(String(file))
+}
 ```
 
-Yields:
+Running that with `node example.js` yields:
 
 ```markdown
 # Hello, world!
 ```
 
-[See **unified** for more examples »][unified]
-
 ## API
 
-[See **unified** for API docs »][unified]
-
 This package exports no identifiers.
 The default export is `remarkStringify`.
 
 ### `unified().use(remarkStringify[, options])`
 
-Configure the `processor` to serialize [**mdast**][mdast] syntax trees to
-markdown.
+Add support for serializing markdown.
+Options are passed to [`mdast-util-to-markdown`][mdast-util-to-markdown]:
+all formatting options are supported.
 
-###### `options`
+##### `options`
```

Comment:
Good to see the options are inlined now. It was always a bit much or confusing to know that if you wanted to change how remark formats the output you had to go remark GH --> remark-stringify --> mdast-util-to-markdown